.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
+[**]
+end_of_line = lf
+insert_final_newline = true
+max_line_length = 80
+
+# Set default charset and indentation for python files
+[{*.py, coalib/**/*.py, ./coala, tests/**/*.py}]
+charset = utf-8
+indent_style = space
+indent_size = 4
+trim_trailing_whitespace = true
+
+# 2 space indentation for yml fies and package.json
+[{package.json,*.yml}]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
```

Comment:
Please insert a new line at the end.

Reply:
Can you help me out coz I have mistakenly deleted the branch. :P

Reply:
I actually have the branch present in local.

Reply:
Then just push your local branch to origin.

Reply:
Sry not in local I have it in origin.

Reply:
Then pull it.

Reply:
Done thanks.

Reply:
Changes made.

Reply:
:+1: